### PR TITLE
SK-266 added Get README and updated CHANGELOG

### PR DESCRIPTION
**Pull request by skyflow-srivyshnavi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -504,4 +504,200 @@ Current the following 5 log levels are supported:
...
+ 'errors':[
+   {
+     'error':{
+       'code':404,
+       'description':'Token for skyflowID  doesn"t exist in vault - Request ID: a8def196-9569-9cb7-9974-f899f9e4bd0a'
+     }
+   }
+ ]
+}
+```
```
Comment (manny-silva) [inline README.md:703]: This content should replace the Get by ID section and have equivalent heading levels.

File: samples/README.md [hunk 2]
```diff
@@ -158,5 +158,55 @@ Replace `credentials` with the content of service account credentials file.
...
+| `<REDACTION_TYPE>`             | One of the four Redaction Types.                        |
+| `<SKYFLOW_ID>`                 | Skyflow Id of the record to be fetched.                 |
+| `<UNQIUE_COLUMN_NAME>`         | Unique column name to fetch the data.                   |
+| `<COLUMN_VALUE>`               | Column value of the corresponding column.               |
+
+#### Run the sample
+
+```bash
+python3 get_sample.py
+```
```
Comment (manny-silva) [inline samples/README.md:188]: Move this content above "Get by ID".

File: samples/README.md [hunk 3]
```diff
@@ -158,5 +158,55 @@ Replace `credentials` with the content of service account credentials file.
...
+| `<TABLE_NAME>`                 | Name of the table to insert data into.                  |
+| `<SKYFLOW_ID>`                 | Skyflow Id of the record to be updated.                 |
+| `<FIELD_NAME>`                 | Name of the column to update data.                      |
+| `<VALUE>`                      | Valid value to update into the corresponding column.    |
+
+#### Run the sample
+
+```bash
+python3 update_sample.py
+```
```
Comment (manny-silva) [inline samples/README.md:212]: Move this content below "Get by ID".

**Review by manny-silva — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (skyflow-srivyshnavi) [reply README.md:901]: made changes

File: samples/README.md (hunk 2 above)
Comment (skyflow-srivyshnavi) [reply samples/README.md]: made changes

File: samples/README.md (hunk 3 above)
Comment (skyflow-srivyshnavi) [reply samples/README.md]: made changes

**Pull request merged**